ctf/org.eclipse.tracecompass.ctf.core/src/org/eclipse/tracecompass/ctf/core/event/types/IntegerDeclaration.java

```diff
@@ -315,6 +318,28 @@ public static IntegerDeclaration createDeclaration(int len, boolean signed, int
         return new IntegerDeclaration(len, signed, base, byteOrder, encoding, clock, alignment, role);
     }
 
+    /**
+     * Create method for CTF2 varints
+     *
+     * @param signed
+     *            Is the integer signed? false == unsigned
+     * @param base
+     *            The base (10-16 are most common)
+     * @param role
+     *            The role of the integer declaration
+     * @param varint
+     *            A boolean indicating if the declaration is a varint
+     * @return IntegerDeclaration
+     *
+     * @since 4.4
```

Comment:
Requires plug-in Bundle-Version to be stepped to 4.5

Reply:
Done 👍